linera-views-derive/src/lib.rs

```diff
@@ -146,16 +146,19 @@ fn generate_view_code(input: ItemStruct, root: bool) -> TokenStream2 {
         quote! {}
     };
 
+    // TODO(XXX): this constraint (`field_constraints`) would be nice to have
     quote! {
-        impl #impl_generics linera_views::views::View<#context> for #struct_name #type_generics
+        impl #impl_generics linera_views::views::View for #struct_name #type_generics
         where
             #(#input_constraints,)*
             #(#context_constraints,)*
-            #(#field_constraints,)*
+            // #(#field_constraints,)*
```

Comment:
what's happening here?

Reply:
Fixed in https://github.com/linera-io/linera-protocol/pull/4031/commits/7196ebd1459d9fd5029c0f2ac0dbb282678858d4.